README.md

```diff
@@ -293,6 +293,23 @@ app.intent("vacation", function(request, response) {
 });
 ```
 
+#### AMAZON Specific Intents
+
+Amazon has specific intents that have to do with basic functionality of your skill that you must add.  Some examples of this are `AMAZON.HelpIntent`, `AMAZON.StopIntent`, and `AMAZON.CancelIntent`.  Here is an example of how you would specify that type of intent.
+
+```javascript
+app.intent('AMAZON.StopIntent',{
+  'slots': {},
+  'utterances': []
+}, function (request, response) {
+  	var stopOutput = 'Stopping your Request and Exiting Skill'
+  	response.say(stopOutput).send()
```

Comment:
Probably should follow the style of the previous example section too, dblock uses double quotes ..